next-packages/core/src/style.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+import * as React from 'react'
+import {
+  getRegisteredStyles,
+  insertStyles,
+  isBrowser,
+  getClassName
+} from '@emotion/utils'
+import { serializeStyles } from '@emotion/serialize'
+import { withCSSContext } from '@emotion/core'
+
+type ClassNameArg =
+  | string
+  | boolean
+  | (() => ClassNameArg)
+  | { [key: string]: boolean }
+  | Array<ClassNameArg>
+
+let classnames = (args: Array<ClassNameArg>): string => {
+  let len = args.length
+  let i = 0
+  let cls = ''
+  for (; i < len; i++) {
+    let arg = args[i]
+    if (arg == null) continue
+
+    let toAdd
+    switch (typeof arg) {
+      case 'boolean':
+        break
+      case 'function':
+        if (process.env.NODE_ENV !== 'production') {
+          console.error(
```

Comment:
is this needed? it's already "next major version of Emotion" :D

Reply:
Nope, I made this a while ago and copied `classnames` from v9.